src/System.Security.Cryptography.Cng/tests/Configurations.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
+      netfx-Windows_NT;
```

Comment:
cc @bartonjs so that he can also help us decide if the way to go is to add HashData members to standard so that Cng can build against netstandard.dll